specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json

```diff
@@ -11,4 +11,4 @@
     "syncToken": "zAJw6V16=Njo1IzUxNjQ2NzM=;sn=5164673"
   },
   "specversion": "1.0"
-}
\ No newline at end of file
+}
```

Comment:
It looks like the generated swaggers were previously missing a trailing newline.  Is this fixed in the latest version of `@azure-tools/typespec-autorest`?

Reply:
These are examples and they were actually even flagged in the PR but that check was overridden. I don't think this is the typespec-autorest issue.

Reply:
I believe the examples are created by typespec-autorest though?  I can test this on the latest version of typespec-autorest to make sure the generated examples contain a trailing newline.

Reply:
Most of these examples are written manually.

Reply:
Maybe let's move these newline additions to a separate PR to keep the diffs clean?

Reply:
I included the newline diff here as well because I wanted the entire repo to be prettier clean as part of the upgrade.